docs/reference/ml/apis/find-file-structure.asciidoc

```diff
@@ -549,7 +549,7 @@ If the request does not encounter errors, you receive the following result:
   "quote" : "\"", <6>
   "timestamp_field" : "tpep_pickup_datetime", <7>
   "joda_timestamp_formats" : [ <8>
-    "YYYY-MM-dd HH:mm:ss"
+    "yyyy-MM-dd HH:mm:ss"
```

Comment:
Please revert all the changes in this file.  This field explicitly stores a Joda timestamp format (because Logstash will still be using Joda even after ES is Joda-free, so it's still useful to know this when creating ingest configs).  The file structure finder reports both timestamp formats.  You can see the corresponding Java timestamp format a couple of lines below on line 555, and that already contains `yyyy` instead of `YYYY`.